flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -436,12 +436,20 @@ public boolean isEnabled() {
 		return !properties.isEmpty();
 	}
 
+	/**
+	 * @deprecated Typo in the method name, use {@link #isColumnFamilyAsVariable()}.
+	 */
+	@Deprecated
```

Comment:
this isn't necessary; the method is not present in 1.9 .

Reply:
Even better :-)